templates/lxd/install.html

```diff
@@ -0,0 +1,232 @@
+          <h3 class="p-heading--5">3.&ensp;Install the OS you'd like to use in your container or VM</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <div class="p-tabs">
+            <div class="p-tabs__list js-tabbed-content" role="tablist" aria-label="Install the OS">
+              <div class="p-tabs__item">
+                <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="container-tab"
+                  id="olm">Container</button>
+              </div>
+              <div class="p-tabs__item">
+                <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="vm-tab" id="vm"
+                  tabindex="-1">VM</button>
+              </div>
+            </div>
+          </div>
+          <div tabindex="0" role="tabpanel" id="container-tab" aria-labelledby="Container">
+            <p>Command:</p>
+            <p>
+            <pre>lxc launch {{"<image_server>:<image_name> <instance_name>"}}</pre>
+            </p>
+            <p>Example:</p>
+            <p>
+            <pre>lxc launch ubuntu:22.04 ubuntu-container</pre>
+            </p>
+            <p>Check the <a href="https://images.linuxcontainers.org/">community image server</a> for other Linux distributions.
+            </p>
+          </div>
+          <div tabindex="0" role="tabpanel" id="vm-tab" aria-hidden="true" aria-labelledby="VM">
+            <p>Command:</p>
+            <p>
+            <pre>lxc launch {{"<image_server>:<image_name> <instance_name>"}} --vm</pre>
+            </p>
+            <p>Example:</p>
+            <p>
+            <pre>lxc launch ubuntu:22.04 ubuntu-container</pre>
+            </p>
+            <p>Check the <a href="https://images.linuxcontainers.org/">community image server</a> for other Linux distributions.
+            </p>
+          </div>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">4.&ensp;Run commands</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>You now have your instance up and running! You're all set to experiment with any
+            commands you need.</p>
+          <p>Command:</p>
+          <p><pre>lxc exec {{"<instance_name> -- <command>"}}</pre></p>
+          <p>Example:</p>
+          <p><pre>lxc exec ubuntu-container -- apt-get update</pre></p>
+          <p>For a list of available commands and options, just run</p>
+          <p><pre>lxc</pre></p>
+        </div>
+      </div>
+      <hr class="p-rule u-hide--small">
+      <p>For other installation options, please <a href="https://documentation.ubuntu.com/lxd/en/latest/installing/">check our documentation</a>.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--25-75 is-split-on-medium">
+    <div class="col">
+      <div class="p-section--shallow">
```

Comment:
These shallow sections sure make the render on the phone look strange. There is large gaps between the heading and content.

Reply:
It's up to @lyubomir-popov  but I think we have this spacing consistently in all headings, in which case I don't think we should change it?